app/parsers/bulkrax/csv_parser.rb

```diff
@@ -74,8 +74,28 @@ def create_collections
       collections.each_with_index do |collection, index|
         next if collection.blank?
         break if records.find_index(collection).present? && limit_reached?(limit, records.find_index(collection))
-
-        new_entry = find_or_create_entry(collection_entry_class, unique_collection_identifier(collection), 'Bulkrax::Importer', collection.to_h)
+        ActiveSupport::Deprecation.warn(
+          'Creating Collections using the collection_field_mapping will no longer be supported as of Bulkrax version 3.0.' \
+          ' Please configure Bulkrax to use related_parents_field_mapping and related_children_field_mapping instead.'
+        )
+
+        ## BEGIN
+        # Add required metadata to collections being imported using the collection_field_mapping, which only have a :title
+        # TODO: Remove once collection_field_mapping is removed
+        metadata = if collection.delete(:from_collection_field_mapping)
```

Comment:
@ShanaLMoore @bkiahstroud can you explain this?

Reply:
There's a new way and an [old way](https://github.com/samvera-labs/bulkrax/wiki/Configuring-Bulkrax#collection-field-mapping-) of importing collections. The new way works, but the old way was failing because it didn't have the minimum required metadata. This conditional checks if we're using the old way of importing collections, and if we are, it adds the missing metadata

Reply:
should have been more specific. I thought that's what adding this "metadata" variable was for...but beforehand it was being added regardless. now it's only being added if we're trying to delete a collection or something?

Reply:
nevvvvvermind. shana and I talked it out. lol